web-registry/src/components/organisms/ProjectTopSection/ProjectTopSection.tsx

```diff
@@ -58,70 +73,68 @@ function ProjectTopSection({
-  const unit = projectSize?.['qudt:unit']?.['@value'];
-  const areaUnit = getAreaUnit(unit as qudtUnit);
-  const creditClass = project?.creditClassByCreditClassId;
-  const creditClassVersion = creditClass?.creditClassVersionsById?.nodes?.[0];
-  const quote = metadata?.['regen:projectQuote'];
-  const glanceText: string[] | undefined =
-    metadata?.['regen:glanceText']?.['@list'];
-  const primaryDescription =
-    metadata?.['regen:landStory'] || metadata?.['schema:description'];
-  const landStewardStoryTitle = metadata?.['regen:landStewardStoryTitle'];
-  const landStewardStory = metadata?.['regen:landStewardStory'];
+  const {
+    videoURL,
+    glanceText,
+    primaryDescription,
+    quote,
+    landStewardPhoto,
+    landStewardStoryTitle,
+    landStewardStory,
+  } = getUnanchoredMetadata(projectPageMetadata);
 
-  const sdgIris = creditClassVersion?.metadata?.['http://regen.network/SDGs']?.[
-    '@list'
-  ]?.map((sdg: { '@id': string }) => sdg['@id']);
   const { data: sdgData } = useSdgByIriQuery({
     client,
     variables: {
       iris: sdgIris,
     },
     skip: !sdgIris,
   });
-  const sdgs = sdgData?.allSdg.map(s => ({
-    title: s.title || '',
-    imageUrl: getSanityImgSrc(s.image),
+  const sdgs = sdgData?.allSdg.map(sdg => ({
+    title: sdg.title || '',
+    imageUrl: getSanityImgSrc(sdg.image),
   }));
 
   const creditClassSanity = findSanityCreditClass({
     sanityCreditClassData,
     creditClassIdOrUrl:
-      creditClass?.onChainId ||
-      creditClassVersion?.metadata?.['http://schema.org/url']?.['@value'] ||
+      creditClass?.onChainId ??
+      creditClassVersion?.metadata?.['http://schema.org/url']?.['@value'] ??
       onChainProjectId?.split('-')?.[0], // if no offChain credit class
   });
 
+  const displayName =
+    projectName ??
+    (onChainProjectId && `Project ${onChainProjectId}`) ??
+    (projectId && toTitleCase(projectId.replace('-', ' '))) ?? // fallback for legacy handles
+    '';
+
   return (
     <Section classes={{ root: classes.section }}>
       <Grid container>
         <Grid item xs={12} md={8} sx={{ pr: { md: 19 } }}>
-          {!metadata && loading ? (
+          {loading ? (
             <Skeleton height={124} />
           ) : (
-            <Title variant="h1">
-              {projectName ?? `Project ${onChainProjectId}`}
-            </Title>
+            <Title variant="h1">{displayName}</Title>
           )}
           <Box sx={{ pt: { xs: 5, sm: 6 } }}>
             <ProjectPlaceInfo
               iconClassName={classes.icon}
-              // TODO Format and show on-chain project location if no off-chain location
-              place={
-                metadata?.['schema:location']?.['place_name'] ||
-                metadata?.['schema:location']?.['geojson:place_name'] ||
-                onChainProject?.jurisdiction
-              }
+              // TODO Format on-chain jurisdiction if no anchored location
```

Comment:
is this todo still needed?

Reply:
I think so, it'd be a nice to have

Reply:
although we do require the project metadata to have a location...